### DISCUSSION TITLE
Typescript testing / jest mock file

### DISCUSSION THREAD

**Discussion by bglaz2** [pmndrs/zustand#1853] [Bug report] [answered]
Hello, are the typescript testing docs up to date? I am trying to use the provided example for jest, creating zustand.ts in the __mocks__ folder, but every time test jest runs I receive this error: 
```TypeError: (0 , _zustand.default) is not a function```

**Comment by dbritto-dev** [comment]
@bglaz2 would you mind sharing use a basic example? you could use codesandbox for that

**Reply by bglaz2** [reply]
Sorry for the late reply! I didn't notice that in the updated version, the import changed from `create` to `{create}`, which is what was causing my issue.

**Reply by dbritto-dev** [reply, marked as answer]
> Sorry for the late reply! I didn't notice that in the updated version, the import changed from `create` to `{create}`, which is what was causing my issue.

Yes, latest version we moved from default export to named export so instead of `import create from 'zustand'` we use `import { create } from 'zustand'`
